### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by MinsGoing** [nju-websoft/MOLE-CIE#1]
According to the given tutorial, the first task trained normally, but when training the second task, the loss did not decrease. Is there anything I should pay attention to?

***** EE-0  predict metrics *****
  epoch                        =        2.0
  predict_eval_rougeL          =    90.1744
  predict_eval_rougeL_for_CL   =    90.1744
  predict_eval_rougeL_for_EE-0 =    90.1744
  predict_exact_match          =    90.1017
  predict_exact_match_for_CL   =    90.1017
  predict_exact_match_for_EE-0 =    90.1017
  predict_gen_len              =   136.7672
  predict_global_step          =       2268
  predict_loss                 =        nan
  predict_rouge1               =    90.1744
  predict_rouge1_for_CL        =    90.1744
  predict_rouge1_for_EE-0      =    90.1744
  predict_runtime              = 0:09:17.95
  predict_samples              =       5112
  predict_samples_per_second   =      9.162
  predict_steps_per_second     =      0.287
  
Second task loss：
{'loss': 20.3797, 'learning_rate': 5e-05, 'epoch': 0.01}                                                                                                                                                                      
{'loss': 22.8125, 'learning_rate': 5e-05, 'epoch': 0.03}                                                                                                                                                                      
{'loss': 25.3969, 'learning_rate': 5e-05, 'epoch': 0.04}                                                                                                                                                                      
{'loss': 25.3969, 'learning_rate': 5e-05, 'epoch': 0.06}                                                                                                                                                                      
{'loss': 26.4062, 'learning_rate': 5e-05, 'epoch': 0.07}                                                                                                                                                                      
{'loss': 26.15, 'learning_rate': 5e-05, 'epoch': 0.09}                                                                                                                                                                        
{'loss': 25.0625, 'learning_rate': 5e-05, 'epoch': 0.1}                                                                                                                                                                       
{'loss': 25.8313, 'learning_rate': 5e-05, 'epoch': 0.12}                                                                                                                                                                      
{'loss': 27.6406, 'learning_rate': 5e-05, 'epoch': 0.13}                                                                                                                                                                      
{'loss': 29.9094, 'learning_rate': 5e-05, 'epoch': 0.15}                                                                                                                                                                      
{'loss': 30.2469, 'learning_rate': 5e-05, 'epoch': 0.16}                                                                                                                                                                      
{'loss': 29.3875, 'learning_rate': 5e-05, 'epoch': 0.18}                                                                                                                                                                      
{'loss': 27.4969, 'learning_rate': 5e-05, 'epoch': 0.19}                                                                                                                                                                      
{'loss': 28.0719, 'learning_rate': 5e-05, 'epoch': 0.21}                                                                                                                                                                      
{'loss': 26.4031, 'learning_rate': 5e-05, 'epoch': 0.22}                                                                                                                                                                      
{'loss': 26.7969, 'learning_rate': 5e-05, 'epoch': 0.24}                                                                                                                                                                      
{'loss': 27.6031, 'learning_rate': 5e-05, 'epoch': 0.25}                                                                                                                                                                      
{'loss': 25.4469, 'learning_rate': 5e-05, 'epoch': 0.27}                                                                                                                                                                      
{'loss': 25.4438, 'learning_rate': 5e-05, 'epoch': 0.28}                                                                                                                                                                      
{'loss': 24.7156, 'learning_rate': 5e-05, 'epoch': 0.29}                                                                                                                                                                      
{'loss': 26.3719, 'learning_rate': 5e-05, 'epoch': 0.31}                                                                                                                                                                      
{'loss': 25.375, 'learning_rate': 5e-05, 'epoch': 0.32}                                                                                                                                                                       
{'loss': 24.9719, 'learning_rate': 5e-05, 'epoch': 0.34}                                                                                                                                                                      
{'loss': 25.2437, 'learning_rate': 5e-05, 'epoch': 0.35}                                                                                                                                                                      
{'loss': 25.1719, 'learning_rate': 5e-05, 'epoch': 0.37}                                                                                                                                                                      
{'loss': 24.6281, 'learning_rate': 5e-05, 'epoch': 0.38}                                                                                                                                                                      
{'loss': 24.0219, 'learning_rate': 5e-05, 'epoch': 0.4}                                                                                                                                                                       
{'loss': 24.2375, 'learning_rate': 5e-05, 'epoch': 0.41}                                                                                                                                                                      
{'loss': 24.1219, 'learning_rate': 5e-05, 'epoch': 0.43}                                                                                                                                                                      
{'loss': 23.5594, 'learning_rate': 5e-05, 'epoch': 0.44}                                                                                                                                                                      
{'loss': 24.2437, 'learning_rate': 5e-05, 'epoch': 0.46}                                                                                                                                                                      
{'loss': 22.6844, 'learning_rate': 5e-05, 'epoch': 0.47}                                                                                                                                                                      
{'loss': 22.75, 'learning_rate': 5e-05, 'epoch': 0.49}                                                                                                                                                                        
{'loss': 21.6187, 'learning_rate': 5e-05, 'epoch': 0.5}                                                                                                                                                                       
{'loss': 22.7875, 'learning_rate': 5e-05, 'epoch': 0.52}                                                                                                                                                                      
{'loss': 21.925, 'learning_rate': 5e-05, 'epoch': 0.55}                                                                                                                                                                       
{'loss': 22.5594, 'learning_rate': 5e-05, 'epoch': 0.56}                                                                                                                                                                      
{'loss': 22.6625, 'learning_rate': 5e-05, 'epoch': 0.58}                                                                                                                                                                      
{'loss': 22.2812, 'learning_rate': 5e-05, 'epoch': 0.59}                                                                                                                                                                      
{'loss': 22.9094, 'learning_rate': 5e-05, 'epoch': 0.6}                                                                                                                                                                       
{'loss': 22.8875, 'learning_rate': 5e-05, 'epoch': 0.62}                                                                                                                                                                      
{'loss': 22.1, 'learning_rate': 5e-05, 'epoch': 0.63}                                                                                                                                                                         
{'loss': 21.9594, 'learning_rate': 5e-05, 'epoch': 0.65}                                                                                                                                                                      
{'loss': 22.1562, 'learning_rate': 5e-05, 'epoch': 0.66}                                                                                                                                                                      
{'loss': 21.5469, 'learning_rate': 5e-05, 'epoch': 0.68}                                                                                                                                                                      
{'loss': 21.7281, 'learning_rate': 5e-05, 'epoch': 0.69}                                                                                                                                                                      
{'loss': 20.7875, 'learning_rate': 5e-05, 'epoch': 0.71}                                                                                                                                                                      
{'loss': 21.9781, 'learning_rate': 5e-05, 'epoch': 0.72}                                                                                                                                                                      
{'loss': 21.3875, 'learning_rate': 5e-05, 'epoch': 0.74}                                                                                                                                                                      
{'loss': 20.9625, 'learning_rate': 5e-05, 'epoch': 0.75}                                                                                                                                                                      
{'loss': 21.35, 'learning_rate': 5e-05, 'epoch': 0.77}                                                                                                                                                                        
{'loss': 21.1562, 'learning_rate': 5e-05, 'epoch': 0.78}                                                                                                                                                                      
{'loss': 20.9406, 'learning_rate': 5e-05, 'epoch': 0.8}                                                                                                                                                                       
{'loss': 21.0344, 'learning_rate': 5e-05, 'epoch': 0.81}                                                                                                                                                                      
{'loss': 21.7453, 'learning_rate': 5e-05, 'epoch': 0.83}                                                                                                                                                                      
{'loss': 21.0297, 'learning_rate': 5e-05, 'epoch': 0.84}                                                                                                                                                                      
{'loss': 21.3844, 'learning_rate': 5e-05, 'epoch': 0.86}                                                                                                                                                                      
{'loss': 20.5781, 'learning_rate': 5e-05, 'epoch': 0.87}                                                                                                                                                                      
{'loss': 20.05, 'learning_rate': 5e-05, 'epoch': 0.88}                                                                                                                                                                        
{'loss': 20.2219, 'learning_rate': 5e-05, 'epoch': 0.9}                                                                                                                                                                       
{'loss': 19.6203, 'learning_rate': 5e-05, 'epoch': 0.91}                                                                                                                                                                      
{'loss': 20.3562, 'learning_rate': 5e-05, 'epoch': 0.93}                                                                                                                                                                      
{'loss': 19.4656, 'learning_rate': 5e-05, 'epoch': 0.94}                                                                                                                                                                      
{'loss': 20.7094, 'learning_rate': 5e-05, 'epoch': 0.96}                                                                                                                                                                      
{'loss': 19.6531, 'learning_rate': 5e-05, 'epoch': 0.97}                                                                                                                                                                      
{'loss': 19.3172, 'learning_rate': 5e-05, 'epoch': 0.99}
{'loss': 20.2469, 'learning_rate': 5e-05, 'epoch': 1.03}                                                                                                                                                                      
{'loss': 20.2906, 'learning_rate': 5e-05, 'epoch': 1.05}                                                                                                                                                                      
{'loss': 19.2797, 'learning_rate': 5e-05, 'epoch': 1.06}                                                                                                                                                                      
{'loss': 20.3, 'learning_rate': 5e-05, 'epoch': 1.08}                                                                                                                                                                         
{'loss': 20.4875, 'learning_rate': 5e-05, 'epoch': 1.09}                                                                                                                                                                      
{'loss': 19.6938, 'learning_rate': 5e-05, 'epoch': 1.11}                                                                                                                                                                      
{'loss': 19.9125, 'learning_rate': 5e-05, 'epoch': 1.12}                                                                                                                                                                      
{'loss': 19.8656, 'learning_rate': 5e-05, 'epoch': 1.14}                                                                                                                                                                      
{'loss': 19.4984, 'learning_rate': 5e-05, 'epoch': 1.15}                                                                                                                                                                      
{'loss': 19.2047, 'learning_rate': 5e-05, 'epoch': 1.16}                                                                                                                                                                      
{'loss': 18.6938, 'learning_rate': 5e-05, 'epoch': 1.18}                                                                                                                                                                      
{'loss': 18.6578, 'learning_rate': 5e-05, 'epoch': 1.19}                                                                                                                                                                      
{'loss': 19.1828, 'learning_rate': 5e-05, 'epoch': 1.21}                                                                                                                                                                      
{'loss': 18.4547, 'learning_rate': 5e-05, 'epoch': 1.22}                                                                                                                                                                      
{'loss': 18.5875, 'learning_rate': 5e-05, 'epoch': 1.24}                                                                                                                                                                      
{'loss': 19.1062, 'learning_rate': 5e-05, 'epoch': 1.25}                                                                                                                                                                      
{'loss': 18.6469, 'learning_rate': 5e-05, 'epoch': 1.27}                                                                                                                                                                      
{'loss': 18.7453, 'learning_rate': 5e-05, 'epoch': 1.28}                                                                                                                                                                      
{'loss': 19.3078, 'learning_rate': 5e-05, 'epoch': 1.3}                                                                                                                                                                       
{'loss': 18.4812, 'learning_rate': 5e-05, 'epoch': 1.31}                                                                                                                                                                      
{'loss': 18.4203, 'learning_rate': 5e-05, 'epoch': 1.33}                                                                                                                                                                      
{'loss': 18.8078, 'learning_rate': 5e-05, 'epoch': 1.34}                                                                                                                                                                      
{'loss': 18.1078, 'learning_rate': 5e-05, 'epoch': 1.36}                                                                                                                                                                      
{'loss': 17.7297, 'learning_rate': 5e-05, 'epoch': 1.37}                                                                                                                                                                      
{'loss': 18.8469, 'learning_rate': 5e-05, 'epoch': 1.39}                                                                                                                                                                      
{'loss': 18.8109, 'learning_rate': 5e-05, 'epoch': 1.4}                                                                                                                                                                       
{'loss': 18.3375, 'learning_rate': 5e-05, 'epoch': 1.42}                                                                                                                                                                      
{'loss': 18.6625, 'learning_rate': 5e-05, 'epoch': 1.43}                                                                                                                                                                      
{'loss': 18.1219, 'learning_rate': 5e-05, 'epoch': 1.44}                                                                                                                                                                      
{'loss': 19.1844, 'learning_rate': 5e-05, 'epoch': 1.46}                                                                                                                                                                      
{'loss': 18.5969, 'learning_rate': 5e-05, 'epoch': 1.47}                                                                                                                                                                      
{'loss': 18.8281, 'learning_rate': 5e-05, 'epoch': 1.49}                                                                                                                                                                      
{'loss': 17.8391, 'learning_rate': 5e-05, 'epoch': 1.5}                                                                                                                                                                       
{'loss': 17.7547, 'learning_rate': 5e-05, 'epoch': 1.52}                                                                                                                                                                      
{'loss': 17.475, 'learning_rate': 5e-05, 'epoch': 1.53}                                                                                                                                                                       
{'loss': 18.1062, 'learning_rate': 5e-05, 'epoch': 1.55}                                                                                                                                                                      
{'loss': 18.2156, 'learning_rate': 5e-05, 'epoch': 1.56}                                                                                                                                                                      
{'loss': 17.6453, 'learning_rate': 5e-05, 'epoch': 1.58}                                                                                                                                                                      
{'loss': 16.8297, 'learning_rate': 5e-05, 'epoch': 1.59}                                                                                                                                                                      
{'loss': 17.8438, 'learning_rate': 5e-05, 'epoch': 1.61}                                                                                                                                                                      
{'loss': 17.1266, 'learning_rate': 5e-05, 'epoch': 1.62}                                                                                                                                                                      
{'loss': 17.6484, 'learning_rate': 5e-05, 'epoch': 1.64}                                                                                                                                                                      
{'loss': 18.7781, 'learning_rate': 5e-05, 'epoch': 1.65}                                                                                                                                                                      
{'loss': 17.4531, 'learning_rate': 5e-05, 'epoch': 1.67}                                                                                                                                                                      
{'loss': 16.9187, 'learning_rate': 5e-05, 'epoch': 1.68}                                                                                                                                                                      
{'loss': 16.6141, 'learning_rate': 5e-05, 'epoch': 1.7}                                                                                                                                                                       
{'loss': 17.825, 'learning_rate': 5e-05, 'epoch': 1.71}                                                                                                                                                                       
{'loss': 17.2094, 'learning_rate': 5e-05, 'epoch': 1.73}                                                                                                                                                                      
{'loss': 16.8734, 'learning_rate': 5e-05, 'epoch': 1.74}                                                                                                                                                                      
{'loss': 17.0719, 'learning_rate': 5e-05, 'epoch': 1.75}                                                                                                                                                                      
{'loss': 16.2, 'learning_rate': 5e-05, 'epoch': 1.77}                                                                                                                                                                         
{'loss': 17.0977, 'learning_rate': 5e-05, 'epoch': 1.78}                                                                                                                                                                      
{'loss': 16.5094, 'learning_rate': 5e-05, 'epoch': 1.8}                                                                                                                                                                       
{'loss': 17.0625, 'learning_rate': 5e-05, 'epoch': 1.81}                                                                                                                                                                      
{'loss': 17.6781, 'learning_rate': 5e-05, 'epoch': 1.83}                                                                                                                                                                      
{'loss': 16.5391, 'learning_rate': 5e-05, 'epoch': 1.84}                                                                                                                                                                      
{'loss': 16.9641, 'learning_rate': 5e-05, 'epoch': 1.86}                                                                                                                                                                      
{'loss': 16.9922, 'learning_rate': 5e-05, 'epoch': 1.87}                                                                                                                                                                      
{'loss': 16.725, 'learning_rate': 5e-05, 'epoch': 1.89}                                                                                                                                                                       
{'loss': 15.8125, 'learning_rate': 5e-05, 'epoch': 1.9}                                                                                                                                                                       
{'loss': 17.3328, 'learning_rate': 5e-05, 'epoch': 1.92}                                                                                                                                                                      
{'loss': 17.0656, 'learning_rate': 5e-05, 'epoch': 1.93}                                                                                                                                                                      
{'loss': 16.6641, 'learning_rate': 5e-05, 'epoch': 1.95}                                                                                                                                                                      
{'loss': 15.8281, 'learning_rate': 5e-05, 'epoch': 1.96}                                                                                                                                                                      
{'loss': 16.8953, 'learning_rate': 5e-05, 'epoch': 1.98}                                                                                                                                                                      
{'loss': 16.125, 'learning_rate': 5e-05, 'epoch': 1.99}
